RELEASE_NOTES.md

```diff
@@ -23,6 +23,8 @@
 
 - The `ConfigManagingActor` now uses `collections.abc.Mapping` as the output sender type. This change indicates that the broadcasted configuration is intended to be read-only.
 
+- The `PowerDistributingActor` is no longer part of the public API.  The `Result` types for the power distribution results are now exposed through the `frequenz.sdk.microgrid.battery_pool` module.
```

Comment:
Technically all the other actors in `frequenz.sdk.actor` are not longer part of the public API either, they were exposed at the package level (`frequenz/sdk/actor/__init__.py`).

Reply:
looks like I missed a lot of stuff.  Updated.